lisa/tools/gcc.py

```diff
@@ -16,6 +16,13 @@ def command(self) -> str:
     def can_install(self) -> bool:
         return True
 
+    def compile(self, file_name: str, out_put: str) -> None:
+        if file_name:
```

Comment:
should it be `output`? The file_name should be always exists. If the output is optional, provide default value on method declaration.

Reply:
gcc test.c => a.out
gcc test.c -o test => test

Reply:
As the logic `if file_name`, the `file_name` is optional? I think the output is optional.